src/App/Pages/Vault/GroupingsPage/GroupingsPage.xaml

```diff
@@ -189,13 +189,17 @@
                         ItemSelected="AccountRow_Selected"
                         BackgroundColor="{DynamicResource BackgroundColor}"
                         VerticalOptions="Start"
-                        RowHeight="{OnPlatform 70, iOS=70, Android=74}">
+                        RowHeight="{OnPlatform 70, iOS=70, Android=74}"
```

Comment:
We should probably add this to Home, Login, and Lock as well (Home for sure has the same problem, the others could potentially at some point)

Reply:
oh ok, didn't think there were any modals there, but yes I'll add this everywhere